index.html

```diff
@@ -1,12 +1,148 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Trekkie</title>
+
 </head>
+
 <body>
-    
+    <section class="home">
+
+        <nav class="nav">
+            <section class="header-logo">
+                <section class="header-svg"><svg xmlns="http://www.w3.org/2000/svg"
+                        xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img"
+                        class="iconify iconify--iconoir" width="32" height="32" preserveAspectRatio="xMidYMid meet"
+                        viewBox="0 0 24 24">
+                        <path fill="none" stroke="#FFEA29" stroke-linecap="round" stroke-linejoin="round"
+                            stroke-width="1.8"
+                            d="m18 10l-3 1.5l-4-3l-1 5.5l3.5 3l.5 4.5m4-13v13M10 17l-2 4.5m.5-13C7 9.5 6 12 6 12l2 1m4-6.5a2 2 0 1 0 0-4a2 2 0 0 0 0 4Z">
+                        </path>
+                    </svg></section>
```

Comment:
If you are using something which is not generic, kindly add comments for better understanding.